### PR TITLE
move PRT header to the top

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

File: opm/simulators/flow/FlowMainEbos.hpp [hunk 1]
```diff
@@ -271,33 +270,9 @@ namespace Opm
             }
         }
 
-    protected:
-        void setupParallelism()
```
Comment (atgeirr) [inline opm/simulators/flow/FlowMainEbos.hpp]: This function seems unchanged, why is it moved?

Comment (totto82) [reply opm/simulators/flow/FlowMainEbos.hpp]: It is the printPRTHeader that is moved to public.

Comment (atgeirr) [reply opm/simulators/flow/FlowMainEbos.hpp]: I see, thanks. Merging!

**Pull request merged**